app/packages/state/src/hooks/useTo.ts

```diff
@@ -2,7 +2,10 @@ import { useCallback, useContext, useTransition } from "react";
 import { RouterContext, RoutingContext } from "../routing";
 
 const goTo = (router: RoutingContext, path: string, state: any) => {
-  router.history.push(path, state);
+  const searchParams = new URLSearchParams(window.location.search);
+  searchParams.delete("view");
```

Comment:
I assume you considered whether there are any consequences of this in non-notebook-screenshot contexts?